x-pack/plugins/cases/public/components/user_actions/comment/alert.test.tsx

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { CommentResponseAlertsType } from '../../../../common/api';
+import { SnakeToCamelCase } from '../../../../common/types';
+import { getRuleId, getRuleName } from './alert';
+import { Ecs } from '../../../containers/types';
+
+describe('alert', () => {
+  describe.each([
+    ['getRuleId', getRuleId],
+    ['getRuleName', getRuleName],
+  ])('%s', (name, funcToExec) => {
+    it('returns the first entry in the comment field', () => {
+      const comment = {
+        rule: {
+          id: ['1', '2'],
+          name: ['1', '2'],
```

Comment:
are `1` and `2` supposed to be numbers or strings that correspond to a name? if such, could you modify these tests with something more alike to real data like `comment name` or a string that doesn't reassemble a number?